docker/blackduck-alert-db/Dockerfile

```diff
@@ -10,9 +10,14 @@ LABEL com.blackducksoftware.integration.alert.vendor="Black Duck Software, Inc."
 
 ENV DOCKER_ENTRYPPOINT_INITD_DIR /docker-entrypoint-initdb.d
 
+USER root
```

Comment:
We don't like to set the user because with openshift the customer can configure the user that runs the container and owns it.  So we don't want to run as the root user.  The recommendation we have had in the past is to not set the user in the container because customer's will want to configure the user due to security policies.